lib/matplotlib/collections.py

```diff
@@ -1934,6 +1934,12 @@ class QuadMesh(Collection):
 
     shading : {'flat', 'gouraud'}, default: 'flat'
 
+    Attributes
+    ----------
+    show_cursor_data : bool
```

Comment:
Do we have a comparable API on anything else?

Reply:
Searching the docs for `cursor_data` suggests we don't.

Reply:
Do we want to make this a general Artist API, and set as appropriate for different artists (like set_snap, set_in_layout, etc)?

Reply:
This sounds like a good idea, but maybe not for a bugfix release.